### DISCUSSION TITLE
Cookies.remove() not working

### DISCUSSION THREAD

**Discussion by Christophvh** [js-cookie/js-cookie#843] [Q&A] [answered]
I'm setting the cookie:
`
Cookies.set('cartId', cart.id, { expires: 30, path: '/' })`

The Cookie is set correctly. 
When i do `Cookies.get('cartId')` it works as expected.
But when I try to delete the cookie it is always undefined and never deletes the cookie.

I tried:
```
Cookies.remove('cartId')
Cookies.remove('cartId' , { path: '/' })
Cookies.set('cartId', cart.id , { expires: 0 })
```

None of the solutions actually delete the cookie. I'm working on 'localhost'. Maybe this is part of the issue?

**Comment by EmilLM** [comment, marked as answer]
I thought I had a similar issue but it was just being set again due to some rerendering.
